src/cart/SConscript

```diff
@@ -22,6 +22,21 @@ SRC = ['crt_bulk.c', 'crt_context.c', 'crt_corpc.c',
        'crt_swim.c', 'crt_tree.c', 'crt_tree_flat.c', 'crt_tree_kary.c',
        'crt_tree_knomial.c', 'crt_hlc.c', 'crt_hlct.c']
 
+LIB_SRC = ['crt_utils.c']
+
+##############################
+# Create cart utility library
+##############################
+def build_utility_library():
+    """Build cart utility library"""
+
+    denv = env.Clone()
+    prereqs.require(denv, 'mercury', headers_only=True)
```

Comment:
(pylint-undefined-variable) Undefined variable 'prereqs'

Reply:
Just pass prereqs into the function to shut this up

Reply:
Wasn't require, no changes.

Reply:
the warning is happening because it doesn't realize prereqs is global.  I think it's cleaner and clearer if you pass it in regardless of whether it is required by SCons or not.